substrate/frame/src/lib.rs

```diff
@@ -342,6 +345,9 @@ pub mod runtime {
 		/// Consider using the new version of this [`frame_construct_runtime`].
 		pub use frame_support::construct_runtime;
 
+		/// Related to runtime contruction.
```

Comment:
I don't understand this comment, I think you can write something better. `BoundedVec` goes far beyond runtime construction. You can also think of moving this to a higher-level prelude.